src/app.py

```
@@ -27,6 +27,7 @@
 from controller.tenant import tenant_controller
 from controller.used_token import used_token_controller
 from controller.user import user_controller
+from controller.parsing import parsing
```

Ruff (0.8.0)

`controller.parsing.parsing` imported but unused

Remove unused import: `controller.parsing.parsing`

(F401)